ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml

```diff
@@ -4170,17 +4170,6 @@ periodics:
   name: periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-4.12-upgrade-from-stable-4.12-aws-ipi-ovn-hypershift-control-plane-p1-f7
   prowjob_defaults:
     tenant_id: gangway-api
-  reporter_config:
```

Comment:
Are those removal expected ?

Reply:
no, they are added manually in prev comments. I just found all report parts are removed automatically after `make jobs`. So I just updated the jobs based on the master branch again.